src/app/client/src/app/plugins/location/components/location-selection/location-selection.component.ts

```diff
@@ -56,10 +58,18 @@ export class LocationSelectionComponent implements OnInit, OnDestroy, AfterViewI
 
   ngOnInit() {
     this.popupControlService.changePopupStatus(false);
+    /**
+      * @description - Here we enable/disable the location popup based on the isvisible value returned from the form config request
```

Comment:
@mithun30052001 refactor this comment Here we enable/disable the location popup based on the isvisible value returned from the form config request